tcms/rpc/api/forms/testrun.py

```diff
@@ -68,7 +68,18 @@ def clean_status(self):
         return data
 
 
-class UpdateExecutionForm(BaseCaseRunForm):
-    tested_by = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
-    build = forms.ModelChoiceField(queryset=Build.objects.all(), required=False)
-    case_text_version = forms.CharField(required=False)
+def validate_case_text_version(value):
```

Comment:
Make this a static method of the form class below b/c it is not used anywhere else.